modules/cnf-about-topology-aware-lifecycle-operator-config.adoc

```diff
@@ -0,0 +1,121 @@
+// Module included in the following assemblies:
+// Epic CNF-2600 (CNF-2133) (4.10), Story TELCODOCS-285
+// * scalability_and_performance/cnf-topology-aware-lifecycle-operator.adoc
+
+:_content-type: CONCEPT
+[id="cnf-about-topology-aware-lifecycle-operator-config_{context}"]
+= About the Topology Aware Lifecycle Operator configuration
+
+The Topology Aware Lifecycle Operator manages the deployment of update policies for one or more SNO clusters in a given duration. The Operator allows you to control the update order of {rh-rhacm}-managed SNO clusters in succession.
+
+The Operator supports the orchestration of the following update scenarios:
+
+* {product-title} y-stream and z-stream updates
+* Day two operations on y-stream and z-streams
+
+The `ClusterGroupUpgrade` CR defines the update that you want to apply to a group of clusters based on the following specifications:
+
+* Clusters in the group
+* Applicable list of managed policies
+* Number of concurrent updates
+* Applicable canary updates
+* Desired {product-title} version
+* Desired Operator version
+
+The `ClusterGroupUpgrade` CR can be in the following states:
+
+`UpgradeNotStarted`:: In this state, the `ClusterGroupUpgrade` CR is created and the `remediationAction` field is set to `inform`.
```

Comment:
```suggestion
`UpgradeNotStarted`:: This is the initial state of the `ClusterGroupUpgrade` CR.
```
There is no remediationAction now. It has been renamed to enable and it can be either true/false, the user can set it up.